src/candidateInfo.css

```diff
@@ -0,0 +1,164 @@
+.modal-candidate-info{
+    display: flex;
+    flex-direction: column;
+    flex-basis: 100%;
+    flex: 2;
+    min-height: 590px;
+    max-height: 590px;
+    overflow-x: hidden;
+    /* padding: 1em; */
+}
+
+.modal-nav-container {
+    min-width: 100%;
+}
+
+.modal-nav {
+    text-align: center;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+}
+
+.close {
```

Comment:
🎨 might wanna make this classname more specific (see comment above re class name collisions)